app/controllers/collection_participants_controller.rb

```diff
@@ -105,7 +105,7 @@ def add
     if @participants_invited.empty? && @participants_added.empty?
       if pseud_results[:banned_pseuds].present?
         flash[:error] =
-            ts("%{name} is currently banned and cannot participate in challenges.",
+            ts("%{name} cannot participate in challenges.",
```

Comment:
Now that this is shorter, we can collapse it to 2 lines to please the Hound
```rb
flash[:error] = ts("%{name} cannot participate in challenges.",
                   name: pseud_results[:banned_pseuds].to_sentence)
```

Reply:
Thanks for that @brianjaustin ! I'll get that in when I push my next commit.